### PR TITLE
Add interact button to play-round

### PR DESCRIPTION
**Pull request by floogulinc**

### CODE REVIEW

**Comment by floogulinc** [comment]
We added the ability to select a bee for yourself on the session test component to help with testing the play round view.

**Comment by floogulinc** [comment]
> The pull request required a lot of set up in firebase to get it working. I would wait until prepare round is merged in. This functionality might not work the same way once there are multiple rounds.

This should work with any specific round, the only thing that's set is that it currently automatically joins the `demo-session`. Also we have provided a UI to setup anything that's needed in the `session-test` page. Arbitrary round creation will not affect how this PR works or how one would have to test it.

**Comment by audreylemeur** [comment]
> > The pull request required a lot of set up in firebase to get it working. I would wait until prepare round is merged in. This functionality might not work the same way once there are multiple rounds.
> 
> This should work with any specific round, the only thing that's set is that it currently automatically joins the `demo-session`. Also we have provided a UI to setup anything that's needed in the `session-test` page. Arbitrary round creation will not affect how this PR works or how one would have to test it.

The UI wasn't working for me or Erik.

**Comment by helloworld12321** [comment]
>>>The pull request required a lot of set up in firebase to get it working. I would wait until prepare round is merged in. This functionality might not work the same way once there are multiple rounds.
>>
>>This should work with any specific round, the only thing that's set is that it currently automatically joins the demo-session. Also we have provided a UI to setup anything that's needed in the session-test page. Arbitrary round creation will not affect how this PR works or how one would have to test it.
>
>The UI wasn't working for me or Erik.

Oh, sorry to hear that. What sort of behavior were you getting?

File: src/app/play-round/play-round.component.html [hunk 1]
```diff
@@ -6,3 +6,12 @@
...
   [alt]="bee.name"
 >
+<button
+  mat-raised-button
+  class="interact-button"
+  color="primary"
+  *ngIf="this.activeRoundMarker$ | async as marker" [disabled]="!marker.active"
+  (click)="clickInteract(marker)"
+>
+  Interact with {{marker.name}}
```
Comment (floogulinc) [inline src/app/play-round/play-round.component.html:16]:
```suggestion
  Visit {{marker.name}}
```

**Pull request merged**